test/plugin/scenarios/grpc-scenario/support-version.list

```diff
@@ -0,0 +1,49 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+1.25.0
+1.24.2
+1.23.1
+1.22.3
+1.21.1
+1.20.0
+1.19.0
+1.18.0
+1.17.1
+1.16.1
+1.15.1
+1.14.0
+1.13.2
+1.12.1
+1.11.0
+1.10.1
+1.9.1
+1.8.0
+1.7.1
+1.7.0
+1.6.1
+1.6.0
+#1.5.0
+#1.4.0
+#1.3.1
+#1.3.0
+#1.2.0
+#1.1.2
+#1.1.1
+#1.0.3
+#1.0.2
+#1.0.1
+#1.0.0
```

Comment:
We don't need comment if you don't support, remove them.

Reply:
Those are very old, even we support part of is, like you said, blocking not support. It is OK to ignore it. I don't think someone is still using this version, even there is, we could simply say, please update the gRPC version.